### PR TITLE
Add Sandbox set_verification_status endpoint

### PR DESCRIPTION
**Pull request by pbernasconi**
Unfortunately we cannot test this automatically – since the /set_verification_status endpoint requires an Item that has gone through the micro-deposit flow in Link, which /sandbox/public_token endpoint doesn't support today.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Instructions for contributing to [plaid-node][1]. A node.js client library for t
 ## Running Tests
 
 ```console
+SECRET=$PLAID_SECRET \
+CLIENT_ID=$PLAID_CLIENT_ID \ 
+PUBLIC_KEY=$PLAID_PUBLIC_KEY \
 $ make test
```
Comment (ravron) [inline CONTRIBUTING.md]: Nit: the `$` in this example (representing the shell prompt) should really precede line 25.

File: README.md [hunk 2]
```diff
@@ -143,6 +143,8 @@ plaidClient.getCategories(cb);
 plaidClient.resetLogin(access_token, cb);
 // Sandbox-only endpoint to trigger a webhook for an Item
 plaidClient.sandboxItemFireWebhook(access_token, webhook_code, cb);
+// Sandbox-only endpoint to set the verfication_status for an Item
+plaidClient.sandboxItemSetVerificationStatus(access_token, webhook_code, cb);
```
Comment (ravron) [inline README.md]: Copy-paste error? The parameters here look like they were copied from `sandboxItemFireWebhook` above.

File: lib/PlaidClient.js [hunk 3]
```diff
@@ -427,7 +427,21 @@ function(access_token, webhook_code, cb) {
       access_token: access_token,
       webhook_code: webhook_code,
     },
-  }, cb, true);
```
Comment (ravron) [inline lib/PlaidClient.js]: Why was this line changed?

Comment (Rockin4Life33) [reply lib/PlaidClient.js]: I don't believe it was.  The three lines before and after changes are shown for context within a diff.

**Comment by skylarmb** [comment]
@pbernasconi bump on this

File: lib/PlaidClient.js (hunk 3 above)
Comment (davidzhanghp) [reply lib/PlaidClient.js]: I have pushed since then but it has actually changed. the boolean parameter is ignored if the cb parameter passed in is a function. it is not good but unfortunately this is how it is. so removing the true is a noop.
